cli/src/main/resources/application.yml

```diff
@@ -216,6 +216,7 @@ kestra:
       max-rows: 5000
     # The expected time for this server to complete all its tasks before initiating a graceful shutdown.
     terminationGracePeriod: 5m
+    taskRestartStrategyOnWorkerFailure: AFTER_TERMINATION_GRACE_PERIOD
```

Comment:
It is a little long
Maybe use `workerTaskRestartStrategy` ? Or resubmitted as I think this is the term that we use elsewhere.